opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -72,9 +95,25 @@ ENV CATALINA_HOME /usr/share/tomcat
 ENV PATH $CATALINA_HOME/bin:$PATH
 WORKDIR $CATALINA_HOME
 
+ARG USER_ID=${TOMCAT_USER_ID:-1000}
+ARG GROUP_ID=${TOMCAT_GROUP_ID:-1000}
+
+# Tomcat user and group created again on actual image definition to run as user.
+RUN    groupadd -r tomcat -g ${GROUP_ID}
+RUN    useradd -u ${USER_ID} -g tomcat -d ${CATALINA_HOME} -s /sbin/nologin \
+        -c "Tomcat user" tomcat
+
+# Change CATALINA_HOME ownership to tomcat user and tomcat group on actual definition
+RUN   chown -R tomcat:tomcat ${CATALINA_HOME}
+RUN   chmod -R 400 ${CATALINA_HOME/conf}
+
 HEALTHCHECK CMD curl --fail http://localhost:8081/healthcheck || exit 1
 
+ENV RUNAS_USER=tomcat
+
+CMD ["whoami"]
+
 EXPOSE 8080 8081 8443
-CMD ["catalina.sh", "run"]
+CMD ["catalina.sh", "run", "-security"]
```

Comment:
From https://tomcat.apache.org/tomcat-9.0-doc/security-manager-howto.html:
> Once you have configured the `catalina.policy` file for use with a SecurityManager, Tomcat can be started with a SecurityManager in place by using the "-security" option

Have we configured the `catalina.policy` file?  If not, we'd be better to leave this out.